src/helper.js

```diff
@@ -144,7 +144,7 @@ function getPackagesPath() {
 }
 
 function getPackageModulePath(packageName) {
-	return path.join(Helper.getPackagesPath(), packageName);
+	return path.join(Helper.getPackagesPath(), "node_modules", packageName);
```

Comment:
For dependencies loaded in local `node_modules`, it is possible to do `require("foo/package")` and `require("foo/package.json")`, which always returns the `package.json` content so there is no need to reconstruct the path.

I was hoping there was a way to tell `require` to use a different search path (in fact, [it's sorta possible with Webpack-specific `require.context` 😩](https://github.com/thelounge/lounge/blob/14847260b699a940627f03293ffb2cd00fc8ddee/client/views/index.js#L3-L7)) then we could just do `require(packageName)` and ``packageName(`${packageName}/package.json`)`` instead of manually constructing this, but so far no luck.